docs/architecture.md

```diff
@@ -52,6 +52,29 @@ SSL termination and front end for the entire infrastructure. No special rules, j
 
 We run a number of Shibboleth service providers, one for each environment.
 
+### Build Artifacts:
+
+| Component | Artifact | Source repo | Notes |
+| --- | --- | --- | --- |
+| UI | Docker image, Nginx w/web app | Source code from `pass-ui`, artifact created in `pass-docker` | Includes NPM package we publish from `pass-emnber-adapter` |
```

Comment:
I prefer linking directly to the projects so instead of `pass-ui` it would be `[pass-ui](https://github.com/eclipse-pass/pass-ui)`, but that won't stop the :+1:

Reply:
Those links would be good for convenience of readers. I'll go ahead and add links to the various relevant GH repos then squash/merge